src/functions-and-arrays.js

```diff
@@ -1,19 +1,61 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1,num2) {
+
+
+
+if(num1>num2){
+  return  num1;
+}else{
+  return num2;
+}
+
+ 
+}
+
 
 
 
 // Iteration #2: Find longest word
+
+
+
+
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(words) {
+  
+  for(let i=0; i<words.length;i++){
+    
+    for(let j=0; word[i].length;j++){
+      if(words[i][j]>10){
+        const result= words[i][j];
+        return result;
+      }
+    }
+  }
+  
+  
+  return result;
+}
```

Comment:
- En los test te dice que `word` is not defined. Has hecho un bucle dentro de otro bucle, y en el segundo intentas acceder a la posición i de `word`, pero `word` no existe, la variable que contiene el array es `words`.
- Dale una vuelta más que hay opciones más sencillas de solucionarlo! (Pista: en cada iteración puedes ir almacenando una palabra solo y solo si es más larga que la que ya estaba almacenada 😉 )

Reply:
recibido gracias.